src/plot.jl

```diff
@@ -88,49 +88,47 @@ Equal to 0 for undirected graphs. Default: `0.1` for the directed graphs
 Angular width in radians for the arrows. Default: `π/9 (20 degrees)`
 
 `linetype`
-Type of line used for edges ("straight", "curve"). Default: "straight"
+Type of line used for edges (:straight, :curve). Default: :straight
 
 `outangle`
 Angular width in radians for the edges (only used if `linetype = "curve`). 
 Default: `π/5 (36 degrees)`
 
+`backgroundc`
+Color for the plot background. Default: `nothing`
+
 """
 function gplot(g::AbstractGraph{T},
     locs_x_in::Vector{R1}, locs_y_in::Vector{R2};
     nodelabel = nothing,
     nodelabelc = colorant"black",
     nodelabelsize = 1.0,
-    NODELABELSIZE = 4.0,
+    max_nodelabelsize = 4.0,
```

Comment:
Totally agree, that we should change this, but I think we should add deprecation warnings for the older functions (that we can eventually remove), or at least produce a meaningful error message.

Reply:
ok. will do